api.md

```diff
@@ -39,10 +39,10 @@
     ```
 
 * `/api/institutions/<institution>/periods/<period>/submissions/<submission>/edits`
-    * `GET` - List of all edits for a given submission, grouped by edit type. Structure: `{"syntactical": {...}, "validity": {..}, "quality": {...}, "macro": {...}}`
+    * `GET` - List of all edits for a given submission, grouped by edit type. Structure: `{"syntactical": {...}, "validity": {..}, "quality": {...}, "macro": {...}, "q029": {...}, "q595": {...}}`
     * `PUT` - Bulk update of edits, used for verification - TODO
 
-* `/api/institutions/<institution>/periods/<period>/submissions/<submission>/edits/syntactial|validity|quality|macro`
+* `/api/institutions/<institution>/periods/<period>/submissions/<submission>/edits/syntactial|validity|quality|macro|q029|q595`
```

Comment:
So I have some thoughts/revisions on this structure:
 - `.../edits/<edit>` should be an endpoint which returns just the `<edit>` specified. The endpoints for q029 and q595 would be captured by this.

- syntactical/validity/quality/macro/ should be part of a querystring with key 'type' so to get all macro edits you'd hit `.../edits?type=macro`

- a `lar` key in the querystring would allow filtering by lar, which could perhaps be expanded to accept a range of values for paging (stay tuned)

- a `groupBy` key in the querystring would support sorting based on lar where appropriate

I'll be working on these ideas in the next few days; out-of-scope for this PR; but thought I should mention it here so we're thinking about things the same way